### ISSUE TITLE
No location info in the screen output or csv download

### ISSUE THREAD

**Issue by jsimons9** [GoMexSI/gomexsi-web#63]
There is no location info (locale name on the spreadsheet) in the screen or csv download.  Not sure if this has been addressed as of yet.

**Comment by jhpoelen** [comment]
@jsimons9 would you like to include the locale name also in this iteration?

**Comment by jsimons9** [comment]
Yes, that is on my list of items to add to the csv.  That could also be added to the query output.

Jim
